sysutils/mpstats/files/mpstats.tcl

```diff
@@ -102,6 +102,22 @@ proc getgccinfo {} {
     }
 }
 
+# extraction of CommandLineTools version
+proc getcltinfo {} {
+    if {[file exists /usr/lib/libxcselect.dylib]} {
+        set pkgname "CLTools_Executables"
+    } else {
+        # Mountain Lion (10.8) and below. Note that we prefer Xcode over CLT for <= 10.8
+        set pkgname "DeveloperToolsCLI"
+    }
+
+    if {![catch {exec pkgutil --pkg-info=com.apple.pkg.${pkgname}} results]} {
+        return lindex $results 3
```

Comment:
Isn't a pair of square brackets missing here?
```suggestion
        return [lindex $results 3]
```
Else this is what I get:
```
> pkgutil --pkg-info=com.apple.pkg.CLTools_Executables
package-id: com.apple.pkg.CLTools_Executables
version: 10.1.0.0.1.1539992718
volume: /
location: /
install-time: 1541237290
groups: com.apple.FindSystemFiles.pkg-group 

> /opt/local/libexec/mpstats-gsoc show | grep clt
    "clt_version": "3"
```

Reply:
Fixed. Thank you for the catch, my bad for only testing on tclsh